.circleci/config.yml

```diff
@@ -8,76 +8,21 @@ buildSteps: &buildSteps
   - run:
       name: Test
       command: npm test --loglevel warn
-  - persist_to_workspace:
-      # Persist all job output, so we can (potentially) use it for deploys
-      root: ../
-      paths:
-        - ./node-*
 
 jobs:
-  node-8:
```

Comment:
I removed node 8 as balena-lint 5.x doesn't support it, the repo will still run fine with it though